arrows/core/detected_object_set_output_kw18.cxx

```diff
@@ -33,12 +33,17 @@
 #include <vital/vital_foreach.h>
 #include <vital/util/tokenize.h>
 
-#include <atomic>
 #include <memory>
 #include <vector>
 #include <fstream>
 #include <time.h>
 
+#if (__GNUC__ == 4 && __GNUC_MINOR__ < 5 )
+  #include <cstdatomic>
```

Comment:
This is failing on Ubuntu with Clang.  See the Travis-CI builds.  Somehow it tries to include cstdatomic when building with Clang.  Maybe these GCC version variables are set if the system has GCC, even if you don't use it?